src/main/java/core/basesyntax/model/User.java

```diff
@@ -58,4 +58,13 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(login, password, age);
     }
+
+    @Override
+    public User clone() {
+        try {
+            return (User) super.clone();
+        } catch (CloneNotSupportedException e) {
+            throw new RuntimeException(e);
+        }
+    }
```

Comment:
let's revert these changes. Actually it's redundant and you don't really need to clone your user